test.sh

```diff
@@ -0,0 +1,13 @@
+curl \
```

Comment:
move the file into a more sensible location

Reply:
done